web/templates/settings/data_exports_admin.hbs

```diff
@@ -44,7 +44,7 @@
                 <th>{{t "Status" }}</th>
                 <th class="actions">{{t "Actions" }}</th>
             </thead>
-            <tbody id="admin_exports_table" data-empty="{{t 'No exports.' }}"></tbody>
+            <tbody id="admin_exports_table" data-empty="{{t 'No exports found.' }}"></tbody>
```

Comment:
Can this be in a separate commit as we are not handling the search filter case here?

Reply:
Added a separate commit. It's not a mandatory change though.

Reply:
So, would be god to have this as a separate commit, in case we do not want to merge it.